site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -4871,7 +4871,7 @@ public void doUser_search(RunData data, Context context) {
 				.getPortletSessionState(((JetspeedRunData) data).getJs_peid());
 
 		// read the search form field into the state object
-		String search = StringUtils.trimToNull(formattedText.escapeHtml(data.getParameters().getString(FORM_SEARCH)));
+		String search = StringUtils.trimToNull(data.getParameters().getString(FORM_SEARCH));
```

Comment:
Does someone see any risk removing this HTML escape? This is causing a text using accents being transformed to "&aacute;". So, the search function will not work for texts using accents.

Reply:
@bgarciaentornos found that the search function for sites is doing it on the same way as my change in my commit: https://github.com/sakaiproject/sakai/commit/3a69b9887fa51ec1327f1c47c6457f1bf74e3777

The data is not being escaped for any possible HTML tag.